### ISSUE TITLE
Chat height

### ISSUE THREAD

**Issue by Anttoneths** [tommyxchow/frosty#205]
custom chat height. for example could set the chat to be 4:3 ratio in the right top corner

**Comment by savaonepunch** [comment]
Yeah, this would be a great feature. I watch forsen a lot for example and the chat covers his face
